stacks/syrah/syrah.yaml

```diff
@@ -615,6 +615,11 @@ external_packages:
         externals:
           - spec: adf@2017.11 %<core_compiler>
             prefix: <external_prefix>/adf
+    - ams:
```

Comment:
It has to be in a list that depends on `intel` `intel-oneapi-mpi`

Reply:
So, we move it to `mpi_packages_intel_stable`?